app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")

```

Comment:
maybe u want to provide some additional info like
"Visitor shoud be vaccinated" etc.